eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/config/WebMvcConfig.java

```diff
@@ -0,0 +1,23 @@
+package hu.flowacademy.meetingorganizer.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@EnableWebMvc
+@Configuration
+public class WebMvcConfig implements WebMvcConfigurer {
```

Comment:
Erre szerintem már nem lesz szükség

Reply:
Dob egy ilyen hibát a Travis: Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.springframework.security.oauth2.client.registration.ClientRegistrationRepository' available, ezt hogy lehetne kijavítani?

Reply:
Töröltem teljesen a WebMvcConfigot, de a Travis hiba még fenn áll ettől függetlenül.